packages/clerk-js/src/ui/utils/passwordUtils.ts

```diff
@@ -29,13 +29,29 @@ type LocalizationConfigProps = {
   locale: string;
   passwordSettings: Pick<PasswordSettingsData, 'max_length' | 'min_length'>;
 };
+
+type ZxcvbnSuggestionKeys = {
+  code: string;
+  message: string;
+};
+
 export const createPasswordError = (errors: ClerkAPIError[], localizationConfig: LocalizationConfigProps) => {
   if (!localizationConfig) {
     return errors[0].longMessage;
   }
 
   const { t, locale, passwordSettings } = localizationConfig;
 
+  if (errors.length && errors[0] && errors[0].code === 'form_password_not_strong_enough') {
+    const message = errors[0].meta?.zxcvbn?.suggestions
+      ?.map((s: ZxcvbnSuggestionKeys) => {
```

Comment:
❓Why do we need an explicit type here ? Should  this  be inferred based on the types you added bellow ?

Reply:
@panteliselef I agree this type doesn’t add much value